tests/client/amo/components/TestAddonDetail.js

```diff
@@ -441,12 +441,15 @@ describe('AddonDetail', () => {
     });
 
     it('prompts you to read many reviews', () => {
+      const i18n = getFakeI18nInst();
       const root = reviewFooterDOM({
         ratingsCount: 5,
+        i18n,
       });
       const footer =
         root.querySelector('.AddonDetail-read-reviews-footer');
       assert.equal(footer.textContent, 'Read all 5 reviews');
+      assert.ok(i18n.ngettext.calledWith('Read %(count)s review', 'Read all %(count)s reviews', 5));
```

Comment:
So do you think it's better to leave the test change out entirely? The no-op ngettext would have passed a test showing the english singular phrase for a single rating.